### PR TITLE
update GDAL to 3.0.3

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

**Comment by fjperini** [comment]
@3nids I will update this later, first it is necessary to make other updates

**Comment by 3nids** [comment]
@fjperini gentle ping.
what other updates are required?

File: Formula/osgeo-gdal.rb [hunk 1]
```diff
@@ -20,8 +20,8 @@ def message
 class OsgeoGdal < Formula
   desc "GDAL: Geospatial Data Abstraction Library"
   homepage "https://www.gdal.org/"
-  url "https://github.com/OSGeo/gdal/releases/download/v3.0.2/gdal-3.0.2.tar.gz"
-  sha256 "787cf150346e58bff0ccf8c131b333139273e35d2abd590ad7196a9ee08f0039"
+  url "https://github.com/OSGeo/gdal/releases/download/3.0.3/gdal-3.0.3.tar.gz"
```
Comment (fbonzon) [inline Formula/osgeo-gdal.rb]:
```suggestion
  url "https://github.com/OSGeo/gdal/releases/download/v3.0.3/gdal-3.0.3.tar.gz"
```

**Comment by 3nids** [comment]
@fjperini this PR has a green light. Please let us know if we can merge or not.

**Comment by luispuerto** [comment]
@fjperini it would be great if we can merge this. I just tried in my computer and it works. I needed to install some R packages.

**Comment by 3nids** [comment]
considering the sillence for about 3 weeks, I'd consider trying ourselves.
any idea which formula would require update before gdal?

**Comment by 3nids** [comment]
@fbonzon funny to meet you here...I was doing my master thesis with Jan @ topo, EPFL when you were there too

**Comment by fjperini** [comment]
sorry for the absence, now if I start with this!

**Comment by fjperini** [comment]
@3nids @luispuerto @fbonzon between today and tomorrow I update to the repository!

**Pull request closed without merging**